### DISCUSSION TITLE
program stalling

### DISCUSSION THREAD

**Discussion by beugo** [AloneLiberty/FlipperNested#14] [Q&A]
Recovering key type B, sector 7
[=] Hardnested attack starting...
[=] ---------+---------+---------------------------------------------------------+-----------------+-------
[=]          |         |                                                         | Expected to brute force
[=]  Time    | #nonces | Activity                                                | #states         | time
[=] ---------+---------+---------------------------------------------------------+-----------------+-------
[=]        0 |       0 | Start using 8 threads                                   |                 |
[=]        0 |       0 | Brute force benchmark: 61 million (2^25.9) keys/s       | 140737488355328 |   27d
[=]        1 |       0 | Using 235 precalculated bitflip state tables            | 140737488355328 |   27d
[=]        6 |     256 | Loading nonces from file                                |     74713776128 | 20min
[=]       10 |     512 | Loading nonces from file                                |     41735979008 | 11min
[=]       11 |     768 | Loading nonces from file                                |     41735979008 | 11min
[=]       12 |    1024 | Loading nonces from file                                |     41735979008 | 11min
[=]       12 |    1280 | Loading nonces from file                                |     41735979008 | 11min
[=]       12 |    1536 | Loading nonces from file                                |     41735979008 | 11min
[=]       14 |    1775 | Apply Sum property. Sum(a0) = 128                       |      3917915904 |   64s
[=]       14 |    1776 | Apply bit flip properties                               |      3917915904 |   64s
[=]       14 |    1777 | Apply bit flip properties                               |      3917915904 |   64s
[=]       14 |    1778 | Apply bit flip properties                               |      3917915904 |   64s
[=]       14 |    1778 | (Ignoring Sum(a8) properties)                           |      3917915904 |   64s

I keep getting to about here and then the program just stays like this for ages so I have to end up using task manager to end the process, control C doesn't work. Can anyone help me fix this?

**Comment by AloneLiberty** [comment]
If your CPU is slow, it might take hour and over. Even on my 5950x it sometimes takes 30 minues (as no SMID and uses 1 core)
